ethereumj-core/src/main/java/org/ethereum/config/net/MainNetConfig.java

```diff
@@ -32,5 +32,6 @@ public MainNetConfig() {
         add(2_463_000, new Eip150HFConfig(new DaoHFConfig()));
         add(2_675_000, new Eip160HFConfig(new DaoHFConfig()));
         add(4_370_000, new ByzantiumConfig(new DaoHFConfig()));
+        add(6_000_000, new ErpConfig());
```

Comment:
Take a look at [JsonNetConfig](https://github.com/ethereum/ethereumj/blob/develop/ethereumj-core/src/main/java/org/ethereum/config/net/JsonNetConfig.java) you might want to handle `ErpConfig` there too